src/app/components/Results/Results.jsx

```diff
@@ -48,17 +48,17 @@ class Results extends React.Component {
 
   getResultsSort() {
     return sortingOpts.map((d, i) => {
-      return (<li role="region" key={i} onClick={e => this.sortResultsBy(e, d.val, d.label)}>
-        {d.label}
+      return (<li role="region" key={i}>
+        <a href="#" onClick={e => this.sortResultsBy(e, d.val, d.label)}>{d.label}</a>
```

Comment:
the example assumed links of the sort:

````html
<a href="?sort=relevance"><strong>relevance</strong></a>
````

where the sorting parameters are deep-linkable (and behave like links)... if that is not the case (i have a preference for links over buttons in general) then we can update the toolkit to the `button` tag or similar... a PR or issue in the toolkit will be welcome